client/src/pages/Programmes/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Flex, Text, SimpleGrid, IconButton } from "@chakra-ui/core";
+import SearchBar from "../../components/SearchBar";
+import ProgrammeCard from "../../components/ProgrammeCard";
```

Comment:
Could you put these components in the `index.js` in the components folder so we can import them like `import {SearchBar, ProgrammeCard}` 

Just helps keeps the imports a bit cleaner :)

Reply:
Yes let me refactor! So do all components have their own `index.js` file with their own folder?

Reply:
Yup, similar to the `AppConfiguration` component I made and how we've been doing pages :)